test/components.test.js

```diff
@@ -174,3 +175,53 @@ describe('<PropertyList />', () => {
     expect(div.props.children[1].key).toBe('link');
   });
 });
+
+describe('<EditableCodeCell />', () => {
+  let component;
+  let onExecute;
+  let onBlur;
+
+  beforeEach(() => {
+    onExecute = jest.fn();
+    onBlur = jest.fn();
+  });
+
+  it('should display the markup and run onBlur', () => {
+    const markup = '[Testing one:`2 + 3` /]';
+    component = mount(
+      <EditableCodeCell markup={markup} onExecute={onExecute} onBlur={onBlur} />
+    );
+
+    const code = component.find('code');
+    const text = code.text();
+    expect(text).toBe(markup);
+
+    const div = component.find('div');
+    div.simulate('click');
+    expect(onBlur).toHaveBeenCalledTimes(0);
+    div.simulate('blur');
+    expect(onBlur).toHaveBeenCalledTimes(1);
+  });
+
+  it('should display markup and execute on shift + enter', () => {
```

Comment:
Nice tests! Could we also add one for editing the markup (and exiting)?

Reply:
Made a test for checking content editable 🍣

Reply:
Thank you!

And editing the markup is proving to be quite tricky and difficult! WIP for it atm!